etc/kayobe/ansible/migrate-rabbitmq-queues.yml

```diff
@@ -0,0 +1,84 @@
+---
```

Comment:
Instead of using ``-ke om_enable_rabbitmq_quorum_queues=true -ke om_enable_rabbitmq_high_availability=false`` and then the debug message at the end, can we add an assert to the start of the playbook that makes sure these are set? This way we enforce that it's set in config.

Reply:
Done, see https://github.com/stackhpc/stackhpc-kayobe-config/pull/1224/files